virtme_ng/run.py

```diff
@@ -874,23 +872,20 @@ def make(self, args):
             target = "bzImage"
             cross_compile = None
             cross_arch = None
-        make_command = MAKE_COMMAND
+        make_command = MAKE_COMMAND.split()
```

Comment:
Please use `shlex.split` or even better, change MAKE_COMMAND to an array, but please do not forget to copy the value before changing.

```
MAKE_COMMAND = [...]

...
make_command = MAKE_COMMAND.copy()
```

Reply:
Yep.

Reply:
thanks, that's what I did: https://github.com/arighi/virtme-ng/compare/14a20c7d06120d24b6d8692e0ece19daeaabc5f8..dc05350469d80bd056dbbcc2134f49b37d4fd9bb